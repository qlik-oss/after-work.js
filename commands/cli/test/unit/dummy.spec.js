describe('dummy', () => {
  it('should ', () => {
    expect(true).to.equal(true);
    expect(true).toMatchSnapshot();
  });
});
