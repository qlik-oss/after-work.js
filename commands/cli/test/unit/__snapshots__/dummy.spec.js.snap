// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`'should ' 1`] = `true`;

exports[`'should ' 2`] = `true`;

exports[`'should ' 3`] = `true`;
